docs/standards/lukso-standard-proposals/01-lsp2-json-schema.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: 'LSP2 - Storage Layout'
+sidebar_position: 3.3
+---
+
+# The Storage Layout - LSP2 (JSON Schema)
+
+:::caution This section is a work in progress.
+:::
+
+:::success Useful Tip
+
+Our [javascript library `erc725.js`](../../tools/erc725js/getting-started.md) makes it easy to read + write data encoded according to the LSP2 Schema, without going through all the encoding complexity manually.
+
+:::
+
+Smart contracts understand only two things: **bytes** and **numbers**. For a smart contract, there is no such concept as a variable. Everything is stored under the smart contract storage as bytes at specific storage slots.
+
+Data in a smart contract is organised between slots. Each data is storaged at a slot number. Let's take this smart contract as an example.
+
+```
+contract Example {
+
+    string name;    // storage slot 1
+    uint256 age;    // storage slot 2
+    string description; // storage slot 3
+
+}
+```
```

Comment:
can you set it as solidity code?

```suggestion
```solidity
contract Example {

    string name;    // storage slot 1
    uint256 age;    // storage slot 2
    string description; // storage slot 3

}
```
```